JetStreamDriver.js

```diff
@@ -1818,6 +1818,28 @@ let BENCHMARKS = [
         deterministicRandom: true,
         tags: ["Default", "Octane"],
     }),
+    new AsyncBenchmark({
+        name: "prismjs-startup",
+        files: [
+            "./startup-helper/StartupBenchmark.js",
+            "./prismjs/benchmark.js",
+        ],
+        preload: {
+            // Use non-minified bundle for better local profiling.
+            // BUNDLE: "./prismjs/dist/bundle.es5.js",
+            BUNDLE: "./prismjs/dist/bundle.es5.min.js",
```

Comment:
Is there a reason we're running the es5 over the es6 bundle? Either way, it seems reasonable to have a  non-Default version of the other one.

Reply:
- added both es5 and es6 variants
- enabled only es6 by default